tests/_core/reindex.yaml

```diff
@@ -0,0 +1,212 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test reindex.
+epilogues:
+  - path: /movies
+    method: DELETE
+    status: [200, 404]
+  - path: /films
+    method: DELETE
+    status: [200, 404]
+  - path: /videos
+    method: DELETE
+    status: [200, 404]
+  - path: /_ingest/pipeline/transform-and-count
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Create a document.
+    path: /{index}/_doc
+    method: POST
+    parameters:
+      index: movies
+    request_body:
+      payload:
+        title: Beauty and the Beast
+        year: 91
+    response:
+      status: 201
+  - synopsis: Refresh the index.
+    path: /{index}/_refresh
+    method: POST
+    parameters:
+      index: movies
```

Comment:
The first 2 chapters should be part of `prologues` if they are not the operations to be tested in this story. According to the name, only `_reindex` operations are being tested in this story right?